niworkflows/workflows/tests/test_gradunwarp.py

```diff
@@ -0,0 +1,45 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+#
+# Copyright 2021 The NiPreps Developers <nipreps@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# We support and encourage derived works from this project, please read
+# about our expectations at
+#
+#     https://www.nipreps.org/community/licensing/
+#
+"""Check the refmap module."""
+import unittest
+from ..gradunwarp import init_gradunwarp_wf
+from ...testing import has_gradunwarp
+from niworkflows.tests.data import load_test_data
+
+
+@unittest.skipUnless(has_gradunwarp, "Needs Gradunwarp")
+def test_gradunwarp(tmpdir, ds000030_dir, workdir, outdir):
+    """Exercise the Gradunwarp workflow."""
+    tmpdir.chdir()
+
+    wf = init_gradunwarp_wf()
+    if workdir:
+        wf.base_dir = str(workdir)
+
+    print(ds000030_dir)
+    wf.inputs.inputnode.input_file = [
+        str(f) for f in (ds000030_dir / "sub-10228" / "anat").glob("*_T1w.nii.gz")
+    ][0]
+    wf.inputs.inputnode.grad_file = load_test_data('gradunwarp_coeffs.grad')
```

Comment:
I think this needs to be a `str`, and you'll get a `Path` by default.

```suggestion
    wf.inputs.inputnode.grad_file = str(load_test_data('gradunwarp_coeffs.grad'))
```